nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({extended:true}));
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/addition", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const addition = `${firstNumber +secondNumber }`
+  res.send(addition)
+});
+
+app.get("/calculator/subtraction", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const subtraction= `${firstNumber - secondNumber }`
+  res.send(subtraction)
+});
+
+app.get("/calculator/multiplication", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const multiplication= `${firstNumber * secondNumber }`
+  res.send(multiplication)
+});
+
+app.get("/calculator/division", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const division= `${firstNumber/secondNumber }`
+  res.send(division)
+});
+
+app.get("/calculator/add", (req, res) => { 
+const firstNumber = parseInt(req.query.first);
+let sum=0;
+ if(req.query.hasOwnProperty("second")){
+     if(Array.isArray(req.query.second)){
+        req.query.second.forEach((n) => {
```

Comment:
Any reason not to use `reduce` here?

Reply:
Nothing.  I am not sure with reduce , so i choose  foreach